.travis.yml

```diff
@@ -2,11 +2,6 @@ language: node_js
 node_js:
   - node
 
-# Only run travis when pushing to master branch (and PRs)
-branches:
-  only:
-    - master
```

Comment:
This MR is into the initial scaffolding branch and this was causing the tests to not run